### PR TITLE
Get V2 metadata and /content api requests are (currently) always articles

### PR DESCRIPTION
**Pull request by matthew-andrews**
@ironsidevsquincy

### CODE REVIEW

File: server/utils/fetch-capi-v2.js [hunk 1]
```diff
@@ -20,7 +19,7 @@ module.exports = function(opts) {
 				if (!response.ok) {
 					logger.warn('Failed getting CAPIv2 content', {
 						uuid: uuid,
-						type: type,
+						type: 'Article',
```
Comment (matthew-andrews) [inline server/utils/fetch-capi-v2.js]: @ironsidevsquincy is there something I've missed?  I think this is (currently) always articles

**Comment by matthew-andrews** [comment]
Gives you V2 metadata:-
screen shot 2015-03-30 at 15 19 21

File: server/utils/fetch-capi-v2.js (hunk 1 above)
Comment (ironsidevsquincy) [inline server/utils/fetch-capi-v2.js]: Image sets - https://github.com/Financial-Times/next-grumman/blob/master/server/transforms/images.js#L12

Can now see in the log when images are failing

Comment (matthew-andrews) [inline server/utils/fetch-capi-v2.js]: ohhh

**Pull request merged**